map_gen/maps/crash_site/commands.lua

```diff
@@ -243,7 +244,7 @@ function Public.control(config)
 
         Server.to_discord_named_raw(map_promotion_channel,
             crash_site_role_mention .. ' **' .. scenario_display_name[config.scenario_name] .. ' has just restarted!!\\n'
-                .. 'Next map: ' .. new_map_name .. '**')
+                .. 'Next map: ' .. scenario_display_name[new_map_name] .. '**')
```

Comment:
In case the lookup fails.
```suggestion
                .. 'Next map: ' .. scenario_display_name[new_map_name] or new_map_name .. '**')
```